docs/library/jesd204/index.rst

```diff
@@ -423,7 +423,6 @@ HDL Example Projects
   - :git-hdl:`Intel A10Gx (RETIRED) <hdl_2021_r1:projects/daq2/a10gx>`
   - :git-hdl:`AMD Xilinx KC705 <projects/daq2/kc705>`
   - :git-hdl:`AMD Xilinx KCU105 <projects/daq2/kcu105>`
-  - :git-hdl:`AMD Xilinx VC707 (RETIRED) <hdl_2018_r2:projects/daq2/vc707>`
```

Comment:
This should not be removed, as it already states it is retired and it points to 2018_R2